### PR TITLE
fix: Header UI

### PR DESCRIPTION
**Pull request by raop155**
# Description

[UI] Update Header Component

image

## Other:

Refactor markup to match desire design

### CODE REVIEW

File: playwright-report/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (valentinoConti) [inline playwright-report/index.html:1]: shouldn't be here i think

File: src/components/molecules/MainContent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import "./styles.scss";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const MainContent = ({ children }: Props) => {
+  return <div className="main-content">{children}</div>;
```
Comment (valentinoConti) [inline src/components/molecules/MainContent/index.tsx:8]: I'm not really sure we need a component just to add two css lines, I rather add them where needed
(I know you are trying to enforce a standard for all the pages contents, not sure if we need that, but if that's the case a mixin would be better 🙌🏼 )

File: playwright-report/index.html (hunk 1 above)
Comment (raop155) [reply playwright-report/index.html]: fixed

File: src/components/molecules/MainContent/index.tsx (hunk 2 above)
Comment (raop155) [reply src/components/molecules/MainContent/index.tsx:8]: Got it, it make sense!

**Review by valentinoConti — APPROVED** [review]

**Pull request merged**